### ISSUE TITLE
Unable to install IKOS 3.1 on Ubuntu 22.04 using Homebrew

### ISSUE THREAD

**Issue by e1guapo** [NASA-SW-VnV/homebrew-core#7]
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Pop
Description:	Pop!_OS 22.04 LTS
Release:	22.04
Codename:	jammy
```
Using a default Homebrew build (using their install instructions from their homepage).
```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

```
$ brew install nasa-sw-vnv/core/ikos
...
==> Installing nasa-sw-vnv/core/ikos
==> python3 -m venv --system-site-packages --without-pip /home/linuxbrew/.linuxbrew/Cellar/ikos/3.1_1/libexec/vendor
==> python3 -m pip --python=/home/linuxbrew/.linuxbrew/Cellar/ikos/3.1_1/libexec/vendor/bin/python install /tmp/ikos--Pygments
==> cmake -G Unix Makefiles -DGMP_ROOT=/home/linuxbrew/.linuxbrew/opt/gmp -DMPFR_ROOT=/home/linuxbrew/.linuxbrew/opt/mpfr -DPP
==> make install
Last 15 lines from /home/dennis/.cache/Homebrew/Logs/ikos/04.make:
/tmp/ikos-20230818-110125-1bwi6kf/ikos-3.1/ar/src/semantic/data_layout.cpp: In member function ‘ikos::ar::ZNumber ikos::ar::{anonymous}::TypeAlignmentVisitor::operator()(ikos::ar::FloatType*) const’:
/tmp/ikos-20230818-110125-1bwi6kf/ikos-3.1/ar/src/semantic/data_layout.cpp:180:41: error: ‘const class ikos::ar::DataLayout’ has no member named ‘floats’
  180 |         find_alignment_info(data_layout.floats, type->bit_width(), abi));
      |                                         ^~~~~~
make[2]: *** [ar/CMakeFiles/ikos-ar.dir/build.make:202: ar/CMakeFiles/ikos-ar.dir/src/semantic/bundle.cpp.o] Error 1
make[2]: *** [ar/CMakeFiles/ikos-ar.dir/build.make:258: ar/CMakeFiles/ikos-ar.dir/src/semantic/data_layout.cpp.o] Error 1
[ 19%] Linking CXX static library libikos-pp.a
[ 19%] Built target ikos-pp-lib
[ 19%] Building CXX object frontend/llvm/CMakeFiles/ikos-pp.dir/src/ikos_pp.cpp.o
make[2]: *** [ar/CMakeFiles/ikos-ar.dir/build.make:244: ar/CMakeFiles/ikos-ar.dir/src/semantic/context_impl.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1497: ar/CMakeFiles/ikos-ar.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 20%] Linking CXX executable ikos-pp
[ 20%] Built target ikos-pp
make: *** [Makefile:146: all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/nasa-sw-vnv/homebrew-core/issues

These open issues may also help:
IKOS build fails on gcc12 https://github.com/NASA-SW-VnV/homebrew-core/issues/6
Failed to install IKOS v3.1 with Homebrew on Ubuntu 20.04 (WSL-2) https://github.com/NASA-SW-VnV/homebrew-core/issues/5
```

**Comment by CIPop** [comment]
Fails for me on Ubuntu 20:

## Ubuntu 20:
```
==> Installing nasa-sw-vnv/core/ikos
==> python3 -m venv --system-site-packages --without-pip /home/linuxbrew/.linuxbrew/Cellar/ikos/3.1_1/libexec/vendor
==> python3 -m pip --python=/home/linuxbrew/.linuxbrew/Cellar/ikos/3.1_1/libexec/vendor/bin/python install /tmp/ikos--Pygments-20231102-12582-1
==> cmake -G Unix Makefiles -DGMP_ROOT=/home/linuxbrew/.linuxbrew/opt/gmp -DMPFR_ROOT=/home/linuxbrew/.linuxbrew/opt/mpfr -DPPL_ROOT=/home/linu
==> make install
Last 15 lines from /home/cristian/.cache/Homebrew/Logs/ikos/04.make:
      |                                ^
/tmp/ikos-20231102-12582-orrm0b/ikos-3.1/core/include/ikos/core/adt/patricia_tree/set.hpp:890:28: error: 'ikos::core::patricia_tree_utils::match_prefix' cannot be used as a function
  890 |   if (m > n && match_prefix(p, q, n)) {
      |                ~~~~~~~~~~~~^~~~~~~~~
/tmp/ikos-20231102-12582-orrm0b/ikos-3.1/core/include/ikos/core/adt/patricia_tree/set.hpp:892:20: error: 'ikos::core::patricia_tree_utils::is_zero_bit' cannot be used as a function
  892 |     if (is_zero_bit(p, n)) {
      |         ~~~~~~~~~~~^~~~~~
make[2]: *** [ar/CMakeFiles/ikos-ar.dir/build.make:188: ar/CMakeFiles/ikos-ar.dir/src/pass/simplify_upcast_comparison.cpp.o] Error 1
make[2]: *** [ar/CMakeFiles/ikos-ar.dir/build.make:202: ar/CMakeFiles/ikos-ar.dir/src/semantic/bundle.cpp.o] Error 1
make[2]: *** [ar/CMakeFiles/ikos-ar.dir/build.make:216: ar/CMakeFiles/ikos-ar.dir/src/semantic/code.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1528: ar/CMakeFiles/ikos-ar.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 16%] Linking CXX static library libikos-pp.a
[ 16%] Built target ikos-pp-lib
make: *** [Makefile:146: all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/nasa-sw-vnv/homebrew-core/issues

These open issues may also help:
IKOS build fails on gcc12 https://github.com/NASA-SW-VnV/homebrew-core/issues/6
Unable to install IKOS 3.1 on Ubuntu 22.04 using Homebrew https://github.com/NASA-SW-VnV/homebrew-core/issues/7
Failed to install IKOS v3.1 with Homebrew on Ubuntu 20.04 (WSL-2) https://github.com/NASA-SW-VnV/homebrew-core/issues/5
```

**Comment by ivanperez-keera** [comment]
This is a duplicate of https://github.com/NASA-SW-VnV/ikos/issues/225.

**Comment by ivanperez-keera** [comment]
Hi everyone! We've just announced a release candidate for IKOS 3.2, and we've included instructions to install it with homebrew. See https://github.com/NASA-SW-VnV/ikos/discussions/253 for details.

**Comment by ivanperez-keera** [comment]
Closed by the solution to #9.
